subscribe.go

```diff
@@ -54,17 +56,73 @@ func (f HandlerFunc) Handle(msg SubMessage) error {
 
 // ----------------------------------------------------------------------------
 
+// SubStart defines starting position to consume messages.
+type SubStart int
+
+// SubStart options.
+const (
+	// Newest tells to start consuming messages from the newest available
+	// (published AFTER subscribing).
+	Newest SubStart = iota
+
+	// Oldest tells to start consuming messages from the oldest available
+	// (published BEFORE subscribing).
+	Oldest
```

Comment:
Should I rename these like `bps.Newest/Oldest` -> `bps.PositionNewest/PositionOldest`?

Reply:
pls